core/market/Cargo.toml

```diff
@@ -23,7 +23,9 @@ ya-service-api = "0.1"
 ya-service-api-interfaces = "0.1"
 ya-service-api-web = "0.1"
 ya-service-bus = "0.4"
+ya-utils-actix = "0.1"
 
+actix = { version = "0.10", default-features = false }
```

Comment:
we need to bump actix to 0.10 in each yagna service

consider re-exporting actix from ya-utils-actix